api_tests/e2e/rfc003/btc_eth/test.js

```diff
@@ -17,8 +17,7 @@ const alice = actor.create("alice", {});
 const bob = actor.create("bob", {});
 
 const alice_final_address = "0x03a329c0248369a73afac7f9381e02fb43d2ea72";
-const bob_final_address =
-    "bcrt1qs2aderg3whgu0m8uadn6dwxjf7j3wx97kk2qqtrum89pmfcxknhsf89pj0";
+const bob_final_address = "mzNFGtxdTSTJ1Lh6fq5N5oUgbhwA7Nm7cA";
```

Comment:
❓would it work with a bech32 address as well?

Reply:
I don't think it works but I moved that at the start so I'll test more and provide exact findings regarding bech32

Reply:
omnicored does not support bech32 https://github.com/OmniLayer/omnicore/blob/master/src/omnicore/doc/release-notes.md#internal-preperations-for-native-segregated-witness-support

So yes, we must use base58 addresses.